README.md

```diff
@@ -1,2 +1,92 @@
-# Hello-world
```

Comment:
Please add a professional README. A README file should have a project description and steps to run the project locally. You may use [this](https://github.com/microverseinc/readme-template) template from Microverse.

Reply:
Thank you for the review I added a new README file.